ivy-data/src/main/java/com/ivy/data/db/migration/Migration124to125_LoanEditDateTime.kt

```diff
@@ -13,7 +13,7 @@ class Migration124to125_LoanEditDateTime : Migration(124, 125) {
         db.execSQL(
             "CREATE TABLE `${LOANS_TEMP_TABLE}` (`name` TEXT NOT NULL, `amount` REAL NOT NULL, `type` TEXT NOT NULL, `color` INTEGER NOT NULL, `icon` TEXT, `orderNum` REAL NOT NULL, `accountId` TEXT, `isSynced` INTEGER NOT NULL, `isDeleted` INTEGER NOT NULL, `dateTime` INTEGER, `id` TEXT NOT NULL, PRIMARY KEY(`id`))"
         )
-        db.execSQL("INSERT INTO `${LOANS_TEMP_TABLE}` SELECT * FROM `${LOANS_TABLE}`")
+        db.execSQL("INSERT INTO `${LOANS_TEMP_TABLE}` (name, amount, type, color, icon, orderNum, accountId, isSynced, isDeleted, dateTime, id ) SELECT name, amount, type, color, icon, orderNum, accountId, isSynced, isDeleted, dateTime, id FROM `${LOANS_TABLE}`")
```

Comment:
Extract `val columns = "name, amount, type, color, icon, orderNum, accountId, isSynced, isDeleted, dateTime, id` and re-use it. This will solve both the Detekt error and also make to code nicer and more stable

Reply:
Done